notebooks/Perform Real-time Inference with a Custom MONAI Bundle.ipynb

```diff
@@ -145,7 +145,7 @@
     "{\n",
     "    \"root_path\": \"https://[your-storage-account-name].blob.core.windows.net/[your-container-name]\",\n",
     "    \"label_key\": [\"bbox\", \"label\"],\n",
-    "    \"data\": [\n",
+    "    \"data\": [Re\n",
```

Comment:
May I ask why is the `Re` added here?

Thanks,
Bin

Reply:
Thank you @binliunls it's a good catch. It must be caused by my thick fingers.